utils/unittest/fixtures.go

```diff
@@ -466,6 +469,47 @@ func HeaderBodyFixture(opts ...func(header flow.HeaderBody)) flow.HeaderBody {
 	return header
 }
 
+// WithRootDefaults zeroes out all parent‐QC fields and enforces root constraints.
+func WithRootDefaults() func(*flow.UntrustedHeaderBody) {
```

Comment:
same here, it's a very specific function, just move it to the package where you are testing it.